### PR TITLE
Added README.

### PR DESCRIPTION
**Pull request by kele**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Very parser.
+Such report.
+Wow.
```
Comment (kele) [inline README:3]: I'm not sure about this one.

File: README [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Very parser.
```
Comment (empijei) [inline README]:
```suggestion
Very report.
Such parser.
```

File: README [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+Such report.
```
Comment (empijei) [inline README]:
```suggestion
```

File: README (hunk 1 above)
Comment (empijei) [reply README:3]: This should probably stay.

File: README [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Author: Parser McParserFace.
+Very report.
+Such parser.
```
Comment (kele) [inline README]:
```suggestion
Very report. Such parser. One line.
```

Comment (kele) [reply README]: Not doing this bro.

**Comment by kele** [comment]
Pls merge

**Pull request closed without merging**